src/sparql/reports/obsoletiontagcheck.sparql

```diff
@@ -0,0 +1,382 @@
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX replaced_by: <http://purl.obolibrary.org/obo/IAO_0100001>
+PREFIX MONDO: <http://purl.obolibrary.org/obo/MONDO_>
+prefix oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>
+
+#this query reports the label and the comment (when available) corresponding to specific Mondo IDs
+
+SELECT ?mondo_id ?label ?comment ?subset ?deprecated  WHERE 
+  {
+ values ?mondo_id {
```

Comment:
If this query should be in the PR, where did these Mondo IDs come from?

Reply:
My goal was to have the query in the repo so we can use it later. If we want to use it again, we would need to replace the MONDO ids with the new obsoletion candidates. Because of this, I didn't think it was necessary to keep the 300+ terms in the query, and I removed them.